src/palindrom.js

```diff
@@ -780,6 +781,16 @@ var Palindrom = (function() {
         );
       }
     });
+    /* when patches are applied, and their `value`s are objects, make sure to pass the proxified version */
```

Comment:
Because `palindrom-client` uses the objects in patches as models. And if they're unproxified, changing them will go unnoticed. Comment this bit and you'll see `palindrom-client` failing.